packages/tools/devtools/devtools-core/src/messaging/Utilities.ts

```diff
@@ -106,18 +106,27 @@ export function handleIncomingMessage(
 ): void {
 	// TODO: remove loggingOptions once things settle.
 
-	if (message === undefined || !isDevtoolsMessage(message)) {
+	const loggingPreamble =
+		loggingOptions?.context === undefined ? "" : `${loggingOptions.context}: `;
+
+	if (message === undefined) {
```

Comment:
This shouldn't be possible per the type system, but this code was already here. If we hit this, we will log an error.